src/controllers/rewrite.ts

```diff
@@ -1,14 +1,34 @@
 import { Body, Controller, Get, Post } from 'amala'
-import RewriteBody from '@/validators/RewriteBody'
+// import { ChatGPTAPI } from 'chatgpt' // Use this import in DEV mode to have types
+import Persona from '@/models/Persona'
+import RewriteBody from '@/validators/RewriteBody.js'
+import dynamicImport from '@/helpers/dynamicImport'
+import env from '@/helpers/env.js'
 
 @Controller('/')
 export default class LoginController {
   @Get('/')
   index() {
-    return 'Nothing to see here, move along.'
+    return 'Nothing to see here 👀'
   }
   @Post('/rewrite')
-  rewrite(@Body({ required: true }) { persona, text }: RewriteBody) {
-    return { text, persona }
+  async rewrite(
+    @Body({ required: true }) { persona, text, textLength }: RewriteBody
+  ) {
+    try {
+      const { ChatGPTAPI } = await dynamicImport('chatgpt')
```

Comment:
So, the `chatgpt` package is ESM package, `commonJS` doesn't support them. I tried converting this repo to `"type": "module"` with all the stuff from [this](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c) article, everything worked except `amala`, because it imports every `controller` (and every controller uses `import` statement in `.js` files, which `amala` module can't handle because it's not ESM module)

We can use next.js or something else, that supports EMS modules, this approach works too, but if you want to change something with types, you must uncomment import above